test/unittests/test_system.py

```diff
@@ -8,9 +8,23 @@ def test_is_running_from_module(self):
         self.assertFalse(is_running_from_module("mycroft"))
         self.assertTrue(is_running_from_module("unittest"))
 
+    @patch("subprocess.Popen")
     def test_ntp_sync(self):
-        # TODO
-        pass
+        from ovos_utils.system import ntp_sync
+        ntp_sync()
+        popen.assert_called_with("service ntp stop", shell=True)
+        popen.assert_called_with("ntpd -gq", shell=True)
+        popen.assert_called_with("service ntp start", shell=True)
+
+    @patch("subprocess.Popen")
+    def test_timesync_sync(self):
+        from ovos_utils.system import timesync_sync
+        timesync_sync()
+        popen.assert_called_with("sudo systemctl stop systemd-timesyncd", shell=True)
+        popen.assert_called_with("sudo systemctl start systemd-timesyncd", shell=True)
+        timesync_sync(False)
+        popen.assert_called_with("systemctl stop systemd-timesyncd", shell=True)
+        popen.assert_called_with("systemctl start systemd-timesyncd", shell=True)
```

Comment:
these tests should move to the PHAL plugin, in here we can test that the correct bus message is emitted